metadata/type/image.yml

```diff
@@ -26,4 +26,16 @@
       
     - name: abstract
       label: Abstract
+
+    - name: location_of_resources
+      label: Resource Location
+      subfields:
+        - name: url
+          label: Url of the Resource
+
+        - name: location_physical
```

Comment:
Remove one of these subfields, so that we can see the difference between subfields and no subfields